components/Sequence/basic_usage.rst

```diff
@@ -0,0 +1,70 @@
+Basic Usage
+===========
+
+SequenceSubjectInterface
+------------------------
+
+Let's see how an exemplary class implementing **SequenceSubjectInterface** should look like.
+
+.. code-block:: php
+
+    <?php
+
+    use Sylius\Component\Sequence\Model\SequenceSubjectInterface;
+
+    class Order implements SequenceSubjectInterface
+    {
+        /**
+         * @var string
+         */
+        private $number;
+
+        /**
+         * {@inheritdoc}
+         */
+        public function getSequenceType()
+        {
+            return 'order';
```

Comment:
Maybe we should use a class constant here?

``const SEQUENCE_TYPE = 'order';``